gensim/models/fasttext.py

```diff
@@ -771,7 +931,11 @@ def load_fasttext_format(cls, model_file, encoding='utf8', full_model=True):
 
         Notes
         ------
-        Due to limitations in the FastText API, you cannot continue training with a model loaded this way.
+        Facebook provides both `.vec` and `.bin` files with their modules.
```

Comment:
this method deserves a good example
- load FB model + infer + train
- load FB without second matrix & infer

Reply:
Done